core/primitives-core/src/account.rs

```diff
@@ -188,20 +188,22 @@ impl Account {
 
 #[cfg(feature = "protocol_feature_add_account_versions")]
 impl BorshDeserialize for Account {
-    fn deserialize(buf: &mut &[u8]) -> Result<Self, Error> {
+    fn deserialize(buf: &mut &[u8]) -> Result<Self, io::Error> {
```

Comment:
With this change, it seems that we don't need to gate this implementation with a feature flag.

Reply:
Without feature Account is struct and have derived BorshDeserialize